modules/javascript/src/main/webapp/javascript/atmosphere.js

```diff
@@ -2748,8 +2755,6 @@
                 if (m.id !== guid) {
                     if (typeof (_request.onLocalMessage) !== 'undefined') {
                         _request.onLocalMessage(m.event);
-                    } else if (typeof (atmosphere.util.onLocalMessage) !== 'undefined') {
```

Comment:
Similar to with `onTransportFailure`, `atmosphere.util` simply does not have a field `onLocalMessage`.